lib/commands/element.js

```diff
@@ -156,6 +159,67 @@ commands.getSize = async function (el) {
   }
 };
 
+commands.getElementContentSize = async function (el) {
+  el = unwrapEl(el);
+  const command = `au.getElement('${el}').childElementsFrames()`;
+  const frames = await this.uiAutoClient.sendCommand(command);
+  const type = await this.getName(el);
+
+  let contentHeight = 0;
+  switch (type) {
+    case "UIATableView": {
+      const firstElementFrame = frames[0];
+      const lastElementFrame = frames[frames.length - 1];
+      contentHeight = lastElementFrame.origin.y + lastElementFrame.size.height - firstElementFrame.origin.y;
+      break;
+    }
+    case "UIACollectionView": {
+      let firstElementFrame = frames[0];
+      let elementsInRow = 0;
+
+      // Get number of elements in a row
+      for (let i = 0; i < frames.length; i++) {
+        const frame = frames[i];
+        const yCoord = frame.origin.y;
+
+        if (yCoord !== firstElementFrame.origin.y) {
+          elementsInRow = i;
+          break;
+        }
+      }
+
+      const spaceBetweenElements = frames[elementsInRow].origin.y - frames[elementsInRow - 1].origin.y - frames[elementsInRow - 1].size.height;
+      const numberOfRows = Math.ceil(frames.length/elementsInRow);
+
+      // Assume that all cells are of the same size
+      contentHeight = numberOfRows * firstElementFrame.size.height + spaceBetweenElements * (numberOfRows - 1);
+      break;
+    }
+    default: {
+      // This method calculates content size only for UIATableView and UIACollectionView. UIAWebView is comming soon.
+      return null;
```

Comment:
it's a bad practice to return null. just throw an exception instead (log.errorAndThrow)